### PR TITLE
validation styling login

### PR DESCRIPTION
**Pull request by FranciscoSenestrari**
Remove console log

Co-authored-by: ReymartinMontes <reymartinmontes@gmail.com>
Co-authored-by: ayeza-zamudio3 <ayezahmvc14@gmail.com>
Co-authored-by: robermancera <robermancera@gmail.com>
Co-authored-by: nicolasjrz <nicolasjrz.dev@gmail.com>

# Description
What did you implemented/Why did you implemented this:
 - Explain the changes you’ve made. 
 - It doesn’t need to be fancy and you don’t have to get to technical, yet. 
 - Just explicit what you have implemented at a high level. 
 - Let the reviewer know the overall effect of the PR.
 - Reference a ticket in your issue tracker. Explain what the change is and then and only then reference the ticket.
 - The “why” is a chance to explain both the engineering goal, but also a some business objective that is satisfied or moved along.
 
 Examples:
 - What?: I've added support for authentication to implement Key Result 2 of OKR1. It includes 
model, table, controller and test. For more background, see ticket #JIRA-123.
 - Why?: These changes complete the user login and account creation experience. See #JIRA-123 for more information.
 
 # Testing
 Help me how can I test or look at the changes
 
 Example:
  - I've added coverage for testing all new methods. I used Faker for a few random user emails and names.
  
  ## Screenshots 
If applicable, include screenshots of the results or screenshots that help to see changes

### CODE REVIEW

**Review by LizyMiGo — APPROVED** [review]
Nice

File: App/src/screen/Login/Login.js [hunk 1]
```diff
@@ -17,6 +19,16 @@ const Login = ({navigation}) => {
   const login = () => {
     console.log(state);
```
Comment (YilKanda) [inline App/src/screen/Login/Login.js:20]: Delete the console.log(), you can use Alert.alert()

File: App/src/screen/Login/Login.js [hunk 2]
```diff
@@ -17,6 +19,16 @@ const Login = ({navigation}) => {
...
   };
+
+  const formik = useFormik({
+    initialValues: initialValues(),
+    validationSchema: Yup.object(validationSchema()),
+    validateOnChange: false,
+    onSubmit: formValue => {
+      console.log('Formulario enviado..');
+      console.log(formValue);
```
Comment (YilKanda) [inline App/src/screen/Login/Login.js:29]: Delete the console.log() of your code in GitHub, you can use Alert.alert()

**Review by YilKanda — CHANGES_REQUESTED** [review]

File: App/src/screen/Login/Login.js (hunk 1 above)
Comment (FranciscoSenestrari) [reply App/src/screen/Login/Login.js:20]: using console.log for compatiblity develop in ios and android but deleting this code

**Review by daniel-yocupicio — DISMISSED** [review]

**Review by LizyMiGo — APPROVED** [review]
Nice

**Review by mrivh — APPROVED** [review]

**Review by GerardoTovar — DISMISSED** [review]

File: App/src/screen/Login/Login.js (hunk 1 above)
Comment (FranciscoSenestrari) [reply App/src/screen/Login/Login.js]: resolve problems and fix

**Review by daniel-yocupicio — APPROVED** [review]

**Review by LizyMiGo — APPROVED** [review]
Nice

**Review by GerardoTovar — APPROVED** [review]

File: App/src/screen/Login/Login.js (hunk 2 above)
Comment (FranciscoSenestrari) [reply App/src/screen/Login/Login.js]: solved problems

**Review by FranciscoSenestrari — COMMENTED** [review]
finished

**Review by daniel-yocupicio — APPROVED** [review]

**Pull request merged**